### PR TITLE
Fix: Telemetry for billing pages

### PR DESCRIPTION
**Pull request by pamelachia**
## What kind of change does this PR introduce?

Page views in billing will be logged to mixpanel

Dependent on this infra PR being deployed first
https://github.com/supabase/infrastructure/pull/13449

### CODE REVIEW

File: studio/components/ui/PageTelemetry.tsx [hunk 1]
```diff
@@ -54,6 +54,13 @@ const PageTelemetry: FC = ({ children }) => {
           language: telemetryProps?.language,
         },
       })
+
+      post(`${API_URL}/telemetry/pageview`, {
```
Comment (kevcodez) [inline studio/components/ui/PageTelemetry.tsx:58]: Just double-checking -we want a call to `page` and to `pageview`?

Comment (kevcodez) [inline studio/components/ui/PageTelemetry.tsx:58]: Assuming `post` is async and we're not waiting to this fails silently?

Comment (pamelachia) [reply studio/components/ui/PageTelemetry.tsx:58]: Yes pageview is a separate controller just for mixpanel

**Review by kevcodez — APPROVED** [review]
One thing to keep in mind, if we merge this now, this works fine on staging, but the production deployment of the dashboard would depend on the API deployment. Ideally, this only gets merged after the next prod deployment of the API.

**Pull request merged**